### PR TITLE
Dsday24 speakers

### PR DESCRIPTION
**Pull request by Ciandelle**
can I get you both to cast an eye over this please - it's holding information only at the moment until we have something more concrete

### CODE REVIEW

**Comment by Ciandelle** [comment]
This is now only awaiting the session info for the content panel as it feels slightly different, and for the closing keynote :)

File: src/community/design-system-day-2024/session-information.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ Set aside all the rigour and embrace the imperfections of the journey to build a
 
 2:45pm to 3pm
 
-### Closing keynote: To be confirmed
+### Closing keynote: Using the Design System to fight the climate emergency
 
 Ishmael Burdeau, Department of Work and Pensions
```
Comment (NoraGDS) [inline src/community/design-system-day-2024/session-information.md:92]: `Department for Work and Pensions`

File: src/community/design-system-day-2024/speaker-information.md [hunk 2]
```diff
@@ -29,11 +29,11 @@ Zainab Daodu is a seasoned Software Engineer, Technical Writer and founder of Wr
 
 ### Leyla Kee-McParlin, Department for Environment, Food & Rural Affairs
 
-Leyla is a Lead Content Designer at Defra, directing a team working on borders and trade services. She's a service assessor, a Civil Service Social Mobility Ambassador, and does voluntary work with schools in the Liverpool City Region.
+Leyla is a Lead Content Designer at DEFRA, directing a team working on borders and trade services. She's a service assessor, a Civil Service Social Mobility Ambassador, and does voluntary work with schools in the Liverpool City Region.
```
Comment (NoraGDS) [inline src/community/design-system-day-2024/speaker-information.md:32]: `Leyla is a Lead Content Designer at Defra, working on borders and trade services. She's a service assessor, a Civil Service Social Mobility Ambassador, and does voluntary work with schools in Liverpool.`

**Review by NoraGDS — APPROVED** [review]

**Review by trang-erskine — APPROVED** [review]
Looks great. Thanks Ciandelle

**Pull request merged**